src/audiomixerboard.h

```diff
@@ -94,27 +98,38 @@ class CChannelFader : public QObject
 
 public slots:
     void OnLevelValueChanged ( int value ) { SendFaderLevelToServer ( value ); }
+    void OnPanValueChanged ( int value )  {SendPanValueToServer ( value ); }
     void OnMuteStateChanged ( int value );
 
 signals:
     void gainValueChanged ( double value );
+    void panValueChanged  (double value  );
     void soloStateChanged ( int value );
 };
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
```

Comment:
This seems to be a merge issue. This check and the #else branch should be deleted.